### PR TITLE
Update copy around updating to the latest build

### LINKED ISSUE #5274: Update copy around updating to the latest build of CocoaPods
We currently do this:

```` ruby
    def verify_compatibility!
      super
      latest_cocoapods_version = metadata.latest_cocoapods_version && Gem::Version.create(metadata.latest_cocoapods_version)
      return unless Config.instance.new_version_message? &&
        latest_cocoapods_version &&
        latest_cocoapods_version > Gem::Version.new(Pod::VERSION)

      rc = latest_cocoapods_version.prerelease?
      install_message = !Pathname(__FILE__).dirname.writable? ? 'sudo ' : ''
      install_message << 'gem install cocoapods'
      install_message << ' --pre' if rc
      message = [
        '',
        "CocoaPods #{latest_cocoapods_version} is available.".green,
        "To update use: `#{install_message}`".green,
        ("[!] This is a test version we'd love you to try.".yellow if rc),
        ("Until we reach version 1.0 the features of CocoaPods can and will change.\n" \
         'We strongly recommend that you use the latest version at all times.'.yellow unless rc),
        '',
        'For more information, see https://blog.cocoapods.org ' \
        'and the CHANGELOG for this version at ' \
        "https://github.com/CocoaPods/CocoaPods/releases/tag/#{latest_cocoapods_version}".green,
        '',
        '',
      ].compact.join("\n")
      UI.puts(message)
    end
    def verify_compatibility!
      super
      latest_cocoapods_version = metadata.latest_cocoapods_version && Gem::Version.create(metadata.latest_cocoapods_version)
      return unless Config.instance.new_version_message? &&
        latest_cocoapods_version &&
        latest_cocoapods_version > Gem::Version.new(Pod::VERSION)

      rc = latest_cocoapods_version.prerelease?
      install_message = !Pathname(__FILE__).dirname.writable? ? 'sudo ' : ''
      install_message << 'gem install cocoapods'
      install_message << ' --pre' if rc
      message = [
        '',
        "CocoaPods #{latest_cocoapods_version} is available.".green,
        "To update use: `#{install_message}`".green,
        ("[!] This is a test version we'd love you to try.".yellow if rc),
        ("Until we reach version 1.0 the features of CocoaPods can and will change.\n" \
         'We strongly recommend that you use the latest version at all times.'.yellow unless rc),
        '',
        'For more information, see https://blog.cocoapods.org ' \
        'and the CHANGELOG for this version at ' \
        "https://github.com/CocoaPods/CocoaPods/releases/tag/#{latest_cocoapods_version}".green,
        '',
        '',
      ].compact.join("\n")
      UI.puts(message)
    end
```

I think realistically we could probably just remove?

``` ruby
        ("Until we reach version 1.0 the features of CocoaPods can and will change.\n" \
         'We strongly recommend that you use the latest version at all times.'.yellow unless rc),
```

### PR DESCRIPTION
**Pull request by neonichu**
Closes #5274 

cc @orta

### CODE REVIEW

File: spec/unit/sources_manager_spec.rb [hunk 1]
```diff
@@ -135,13 +135,6 @@ module Pod
         UI.output.should.match /CocoaPods 999.0 is available/
       end
 
-      it 'recommends the user to use the latest stable version' do
```
Comment (neonichu) [inline spec/unit/sources_manager_spec.rb]: :joy: that we called them "stable" here before 1.0

Comment (k0nserv) [inline spec/unit/sources_manager_spec.rb]: Hehe that's my choice of wording if I recall :)

**Comment by orta** [comment]
Perhaps an integration spec relied on this?

```
Bacon::Error: File comparison error `execution_output.txt` for install_vendored_dynamic_framework:
--- DIFF -----------------------------------------------------------------------
      
        git checkout -b new_branch_name
      
     Checking out files: 100% (630/630), done.
  > Running prepare command
    $ BASH_BIN -c  set -e sh build.sh cocoapods-setup without-core
 
--- END ------------------------------------------------------------------------
```

**Comment by neonichu** [comment]
I think this is a recurring CI issue, running the specs again.

**Comment by segiddins** [comment]
Yeah, it's a recurring issue. I've made some PRs to CLIntegracon that should either help with the issue or help with diagnosing it

**Comment by segiddins** [comment]
👍🏻

**Pull request merged**